repo_health/index/static/package.json

```diff
@@ -40,6 +40,7 @@
   "scripts": {
     "prestart": "npm prune && npm install",
     "start": "node webpack.server.js",
+    "dist": "node ./node_modules/webpack/bin/webpack --env prod -p --config webpack.config.js",
```

Comment:
`npm run dist` will put app.js and vendor.js in the dist folder in a production ready status :smile:.